### PR TITLE
PR to trigger a CI run for testing purposes

### PR DESCRIPTION
**Pull request by Tieske**

### CODE REVIEW

File: kong/kong.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
--- Kong, the biggest ape in town
+-- Kong, the biggest ape in town 1
```
Comment (thibaultCha) [inline kong/kong.lua]: `git commit --allow-empty` can be neat for that, avoiding to do painful little changes to force one.

Comment (thibaultCha) [inline kong/kong.lua]: Just a tip, I know this doesn't have for purpose to be merged.

Comment (Tieske) [inline kong/kong.lua]: Didn't know that one. :+1:

Comment (ahmadnassri) [inline kong/kong.lua]: you can manually trigger a CI service from the repo without committing:

image

Comment (thibaultCha) [inline kong/kong.lua]: But from there you cannot chose which branch to trigger the build from, which is useless.

Comment (ahmadnassri) [inline kong/kong.lua]: true, that part you'll have to do on the tool itself, if travis, you can use ENV variables to limit it to a branch ...

**Comment by thibaultCha** [comment]
This was only to trigger a job right? Has any work been done that needs to be merged here? If not, let's close it, otherwise, pick the necessary changes and open a new PR with them.

**Comment by Tieske** [comment]
Yep, it was. I already cherry picked some stuff that went into other PRs.

I'll check what's left. At least one item is a paralleled test run on circleCI. Should save appr 1 minute  on the overall run.

**Pull request closed without merging**